tesseract_task_composer/planning/CMakeLists.txt

```diff
@@ -94,8 +94,8 @@ endif()
 if(tesseract_motion_planners_trajopt_ifopt_FOUND)
   list(APPEND FACTORIES_SOURCE_FILES src/factories/planning_task_composer_trajopt_ifopt_plugin_factory.cpp)
   list(APPEND FACTORIES_SOURCE_LINK_LIBRARIES tesseract::tesseract_motion_planners_trajopt_ifopt)
-  list(APPEND LINUX_DEPENDS "${TESSERACT_PACKAGE_PREFIX}tesseract-motion-planners-trajopt-ifopt")
-  list(APPEND WINDOWS_DEPENDS "${TESSERACT_PACKAGE_PREFIX}tesseract-motion-planners-trajopt-ifopt")
+  list(APPEND LINUX_DEPENDS "${TESSERACT_PACKAGE_PREFIX}tesseract-motion-planners-trajopt_ifopt")
+  list(APPEND WINDOWS_DEPENDS "${TESSERACT_PACKAGE_PREFIX}tesseract-motion-planners-trajopt_ifopt")
```

Comment:
I think this change is correct for the current version of RICB, but I think we should modify the `cpack_component` macro to also replace underscores in component names, which it currently does not. This seems like a non-standard name

Reply:
See [here](https://github.com/ros-industrial/ros_industrial_cmake_boilerplate/pull/99) for the proposed fix